doc/content/reference/telemetry/cli.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "CLI Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the CLI's user"
+aliases: "/telemetry/cli"
+---
+
+{{% tts %}}\'s [CLI]({{<ref "/reference/cli">}}) collects **completely anonymous** telemetry data, which is used to help us understand how users interact with our product. For example, the current data helps to have a better insight on what type of machines the CLI is more commonly used. Telemetry collection is enabled by default and does not collect personally identifiable information. The following section section explains what information is collected and how it is anonymised.
+
+## Identification
+
+When the CLI starts for the first time on a new machine or upon clearing the system cache, it creates a file on the system's cache folder with the following data:
+
+```yaml
+uid: 00000000-0000-0000-0000-000000000000
+last_sent: 2023-08-02T16:45:53.742417-03:00
+```
+
+The `uid` value is an UUID generated according to the specifications made on [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122) and [DCE 1.1](https://pubs.opengroup.org/onlinepubs/9696989899/toc.htm).
+
+## Types of telemetry data
+
+Below is an JSON object with an example of a full telemetry data message that is sent from the CLI:
+
+```json
+{
+  "uid": "00000000-0000-0000-0000-000000000000",
+  "os": {
+    "operating_system": "linux",
+    "arch": "amd64",
+    "binary_version": "3.27.0-dev",
+    "golang_version": " go1.20.7"
+  },
+  "cli": {}, // An empty object for this field indicates that this data was sent by a CLI instance
+  "entities_count": null
+}
+```
+
+{{< note >}}
+All new data collected will be added below with the respective version in which the field was introduced. 
+{{</ note >}}
+
+List of fields and its purpose on by version:
+
+{{< new-in-version "3.27.1">}}
+- `uid`: universal unique identifier attached to telemetry data sent by the CLI.
+- `os`: basic information about the operating system of the machine which executed the CLI.
+- `cli`: signals to the telemetry receptor that the sender should be sending only telemetry data.
+
+## Disable telemetry
+
+The telemetry is enabled by default and in order to disable it, set `telemetry.enable` to `false` in the CLI configuration.
+
+Example of the configuration file with telemetry disabled:
+```yaml
+telemetry:
+  enable: false
+```
+
+## Visualization
+
+The visualization of the usage data collected is displayed at `https://telemetry.thethingsstack.io`.
```

Comment:
```suggestion
Check out [Telemetry Dashboard](https://telemetry.thethingsstack.io) to see visual representation of collected data.
```

Reply:
It would be good if you could shortly describe what can we see there, at least 2-3 important graphs